hms-app/src/actions/ParticipantActions.tsx

```diff
@@ -0,0 +1,42 @@
+const api_id = process.env.REACT_APP_API_ID
+
+export const deleteParticipant = (participantID: string) => {
+  return fetch(
+    `http://localhost:4566/restapis/${api_id}/local/_user_request_/participant/${participantID}`,
+    {
+      method: 'DELETE',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+    }
+  )
+    .then((response) => {
+      return response.json()
+    })
+    .catch((err) => console.log(err))
+}
+
+export const createHackathonParticipant = (props: {
+  userId: string
+  hackathonId: string
+}) => {
```

Comment:
the name of this function could be shortened to just `createParticipant` you don't prefix anything with `Hackathon` anywhere else that I can see. It will shorten overall line length this way